src/renderer/dx/DxRenderer.hpp

```diff
@@ -294,8 +254,6 @@ namespace Microsoft::Console::Render
 
         [[nodiscard]] HRESULT _CopyFrontToBack() noexcept;
 
-        [[nodiscard]] HRESULT _EnableDisplayAccess(const bool outputEnabled) noexcept;
-

```

Comment:
No one in this project calls `Disable()`, so there's not much `_EnableDisplayAccess` can do if no one ever will pass it `false`. 😄 The reason we have this is - I assume - because `DxEngine` originates as a copy of `WddmConEngine` which does have to call the rather important function `WDDMConEnableDisplayAccess`.
But for our `DxEngine` which I suppose won't implement WDDM any time soon, this doesn't do anything, so I removed it, allowing me to add 1 function less to the new `IRenderEngine` interface.

Reply:
You know what... I've just reverted DxEngine entirely now. ^^